tests/service/predictors/srl_test.py

```diff
@@ -46,4 +46,11 @@ def test_cpu_vs_gpu(self):
             prediction_cpu = predictor_cpu.predict_json({"sentence": sentence})
             prediction_gpu = predictor_gpu.predict_json({"sentence": sentence})
 
-            assert prediction_cpu == pytest.approx(prediction_gpu)
+            assert set(prediction_cpu.keys()) == {"verbs"}
```

Comment:
originally the idea was that the weights were from running the same experiment on CPU and GPU, to check that we got the same results. we've messed with these files so much that it is lucky they're passing, and possibly they won't at some point. should I just get rid of this test?